File: 01_mobile_app/front/flutter/app00/lib/model/food_shop.dart
import 'package:flutter/material.dart';

class FoodShop {
  final String id;
  final String name;
  final String? genre;
  final String? lastUsedDate;
  final String? rating;
  final String? memo;

  FoodShop({
    required this.id,
    required this.name,
    this.genre,
    this.lastUsedDate,
    this.rating,
    this.memo,
  });
}

File: 01_mobile_app/front/ionic/app00/src/app/model/FoodShop.ts
export class FoodShop {

  private id: string;
  private name: string;
  private genre: string;
  private lastUsedDate: string;
  private rating: number;
  private memo: string;

  constructor(id: string, name: string, genre: string, rating: number, lastUsedDate: string, memo: string) {
    this.id = id;
    this.name = name;
    this.genre = genre;
    this.lastUsedDate = lastUsedDate;
    this.rating = rating;
    this.memo = memo;
  }

}
